torch/jit/_builtins.py

```diff
@@ -115,7 +115,7 @@ def _get_builtin_table():
     def register_all(mod):
         for name in dir(mod):
             v = getattr(mod, name)
-            if callable(v) and not _is_special_functional_bound_op(v):
+            if callable(v) and not _is_special_functional_bound_op(v) and "no_grad" not in name:
```

Comment:
We should find a cleaner way to do this

Reply:
You should be able to check via identity here, similar to how https://github.com/pytorch/pytorch/blob/3ec5adf3360cbb721d54b6fec61a9e44b097c3e5/torch/jit/_builtins.py#L101 is directly comparing if the objects are the same

Reply:
Done.